Documentation/gitcvs-migration.txt

```diff
@@ -139,7 +139,7 @@ Larger projects or remote repositories may take longer.
 
 The main trunk is stored in the Git branch named `origin`, and additional
 CVS branches are stored in Git branches with the same names.  The most
-recent version of the main trunk is also left checked out on the `master`
+recent version of the main main is also left checked out on the `main`
```

Comment:
Is this correct?

Reply:
Good catch, this is likely a bit too aggressive. "the main trunk" should probably remain.

Reply:
Yep, this is a left-over from the original original patch that renamed `master` to `trunk`. I fixed this via my latest force-push.